src/core/stream.c

```diff
@@ -659,6 +659,22 @@ QuicStreamParamSet(
         break;
     }
 
+   case QUIC_PARAM_STREAM_RELIABLE_OFFSET:
+
+        if (BufferLength != sizeof(uint64_t) || Buffer == NULL) {
+            Status = QUIC_STATUS_INVALID_PARAMETER;
+            break;
+        }
+
+        if (!Stream->Connection->State.ReliableResetStreamNegotiated || *(uint64_t*)Buffer > Stream->QueuedSendOffset) {
```

Comment:
Generally, we try to keep lines under 80 characters long.
```suggestion
        if (!Stream->Connection->State.ReliableResetStreamNegotiated ||
            *(uint64_t*)Buffer > Stream->QueuedSendOffset) {
```